From 05a4c7e97ac64299a971c33fa392297e1021ce3e Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Sun, 20 Feb 2022 14:47:24 +0000
Subject: [PATCH] webos: fix python syntax

Fixes:
[6/37621] ACTION //webos/install/app_shell:app_shell_scripts(//build/toolchain/cros:target)
FAILED: image/usr/bin/app-shell/run_app_shell
/OE/lge/build/webosose/kirkstone/BUILD/work/qemux86_64-webos-linux/webruntime/91.0.4472.114-11-r52.4/recipe-sysroot-native/usr/bin/python3-native/python3 ../../git/src/webos/install/install.py /OE/lge/build/webosose/kirkstone/BUILD/work/qemux86_64-webos-linux/webruntime/91.0.4472.114-11-r52.4/git/src/webos/install/app_shell/run_app_shell /OE/lge/build/webosose/kirkstone/BUILD/work/qemux86_64-webos-linux/webruntime/91.0.4472.114-11-r52.4/build/Release/image/usr/bin/app-shell/run_app_shell
  File "/OE/lge/build/webosose/kirkstone/BUILD/work/qemux86_64-webos-linux/webruntime/91.0.4472.114-11-r52.4/build/Release/../../git/src/webos/install/install.py", line 19
    os.chmod(os.path.join(dst, src) if os.path.isdir(dst) else dst, 0755)
                                                                    ^
SyntaxError: leading zeros in decimal integer literals are not permitted; use an 0o prefix for octal integers

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 src/webos/install/install.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/webos/install/install.py b/src/webos/install/install.py
index a7b658b438..151afeabce 100644
--- a/src/webos/install/install.py
+++ b/src/webos/install/install.py
@@ -16,7 +16,7 @@ def main(*files):
             continue
 
         shutil.copy(src, dst)
-        os.chmod(os.path.join(dst, src) if os.path.isdir(dst) else dst, 0755)
+        os.chmod(os.path.join(dst, src) if os.path.isdir(dst) else dst, 0o755)
 
 
 if  __name__ == "__main__":
